ddp-workspace/projects/ddp-sdk/src/lib/components/user/participantProfile.component.ts

```diff
@@ -29,7 +30,7 @@ import { mergeMap, tap } from 'rxjs/operators';
     <button mat-menu-item *ngFor="let participant of participants" (click)="select(participant.userGuid)">
         <mat-icon *ngIf="isSelected(participant.userGuid); else elseBlock">radio_button_checked</mat-icon>
         <ng-template #elseBlock><mat-icon>radio_button_unchecked</mat-icon></ng-template>
-        {{ participant.alias }}
+        {{ participant.userProfile.firstName }} {{ participant.userProfile.lastName }}
```

Comment:
Can we always expect that userProfile property will be set? (I don't know the answer).
If not sure or don't know then try to be defend for undefined or null condition here.